modules/axruntime/src/mp.rs

```diff
@@ -8,10 +8,11 @@ static mut SECONDARY_BOOT_STACK: [[u8; TASK_STACK_SIZE]; SMP - 1] = [[0; TASK_ST
 
 static ENTERED_CPUS: AtomicUsize = AtomicUsize::new(1);
 
+#[allow(clippy::absurd_extreme_comparisons)]
 pub fn start_secondary_cpus(primary_cpu_id: usize) {
```

Comment:
`primary_cpu_id` should be a logic CPU ID, the hard ID should be only appear on a PSCI call.

BTW, is the `cpu_id` passed in `axhal::rust_entry` a hard ID? Is it always 0?

Reply:
axhal::rust_entry cpu_id is hard id，in phytium pi primary cpu is 200, then 0, 100, 201 start.

Reply:
Maybe we can convert hard ID to logic ID in `axhal::rust_entry`? Then we can get the rid of the `cpu_hard_id_to_logic_id` API.